File: uri/uri_java/iniciante/p1021.java
package iniciante;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class p1021 {
    
    public static void main(String args[]) throws IOException {
        InputStreamReader entrada = new InputStreamReader(System.in);
        BufferedReader leitura = new BufferedReader(entrada);
        
        double valor;
        int i, k;
        int cedula[] = {100,50,20,10,5,2};
        double moeda[] = {1.0,0.5,0.25,0.1,0.05,0.01};
        
        valor = Double.parseDouble(leitura.readLine());
        
        System.out.println("NOTAS:");
        for (i = 0; i < 6; i++) {
            k = (int) (valor/cedula[i]);
            valor%=cedula[i];
            System.out.println(k+" nota(s) de R$ "+cedula[i]+".00");
        }
        
        System.out.println("MOEDAS:");
        k = (int) (valor/moeda[0]);
        System.out.println(k+" moeda(s) de R$ 1.00");
        valor %= moeda[0];
        k = (int) (valor / moeda[1]);
        System.out.println(k+" moeda(s) de R$ 0.50");
        valor %= moeda[1];
        k = (int) (valor / moeda[2]);
        System.out.println(k+" moeda(s) de R$ 0.25");
        valor %= moeda[2];
        k = (int) (valor / moeda[3]);
        System.out.println(k+" moeda(s) de R$ 0.10");
        valor %= moeda[3];
        k = (int) (valor / moeda[4]);
        System.out.println(k+" moeda(s) de R$ 0.05");
        valor %= moeda[4];
        k = (int) (valor / moeda[5]);
        System.out.println(k+" moeda(s) de R$ 0.01");
        valor %= moeda[5];
        System.out.println();
    }
}

File: uri/uri_python/iniciante/p1021.py
#coding: utf-8

valor = float(input())
cedula = [100, 50, 20, 10, 5, 2]
moeda = [1, 0.5, 0.25, 0.10, 0.05, 0.01]

print("NOTAS:")
for i in range(6):
    k = valor / cedula[i]
    valor %= cedula[i]
    print("%d nota(s) de R$ %.d.00" % (k, cedula[i]))

print("MOEDAS:")
k = valor / moeda[0]
print("%d moeda(s) de R$ 1.00" % k)
valor %= moeda[0]
k = valor / moeda[1]
print("%d moeda(s) de R$ 0.50" % k)
valor %= moeda[1]
k = valor / moeda[2]
print("%d moeda(s) de R$ 0.25" % k)
valor %= moeda[2]
k = valor / moeda[3]
print("%d moeda(s) de R$ 0.10" % k)
valor %= moeda[3]
k = valor / moeda[4]
print("%d moeda(s) de R$ 0.05" % k)
valor %= moeda[4]
k = valor / moeda[5]
print("%d moeda(s) de R$ 0.01\n" % k)
valor %= moeda[5]


'''
for i in range(6):
    k = valor / moeda[i]
    valor %= moeda[i]
    print("%d moeda(s) de R$ %1.2f" % (k, moeda[i]))
'''
